models/remoteworkitem.go

```diff
@@ -0,0 +1,96 @@
+package models
+
+import (
+	"fmt"
+	"strconv"
+)
+
+const (
+	ProviderGithub string = "github"
+)
+
+// RemoteWorkItem represents a work item as it is stored in the database
+type RemoteWorkItem struct {
+	Fields map[string]interface{}
+}
+
+// WorkItemMap : holds the mapping between Remote key and local key in a WI
+type WorkItemMap map[string]interface{}
```

Comment:
This is the WorkItemMap, why map[string]interface{} as oppose to map[string]string ?